js/main.js

```diff
@@ -1,3 +1,4 @@
-import {createPhotos} from './mockPhotos.js';
-createPhotos();
+import './mockPhotos.js';
```

Comment:
Идея main.js в целом такая: ты импортируешь интерфейс (одну или несколько функций) из основных модулей, например createPhotos
```suggestion
import {createPhotos} './mockPhotos.js';
```